OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -212,6 +212,15 @@ public AssetBrowserLogic(Widget widget, Action onExit, World world)
 				prevButton.IsVisible = () => !isVideoLoaded;
 			}
 
+			var node = logic.Value.Nodes.FirstOrDefault(x => x.Key == "SupportedFormats");
+			if (node != null)
+			{
+				var value = node.Value.Value.Replace(" ", "");
+				allowedExtensions = value.Split(',');
+			}
+			else
+				allowedExtensions = new string[0];
```

Comment:
OK, that was my initial thinking as well. I just figured people will be against that. I'll remove the fallback value then.